src/styles/index.scss

```diff
@@ -0,0 +1,38 @@
+@import "./blocks/product_card/card";
+@import "./blocks/product_card/stars";
+@import "./blocks/menu/style.css";
```

Comment:
file system should looks different 

:file_folder: blocks

-     header.scss
-     menu.scss
-     card.scss
-     stars.scss

:file_folder: utils

-     _variables.scss
-     _reset.scss

main.scss //add here all imports and firstly add import of variables and reset